config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        "required": [
+                            "inline"
+                        ]
+                    },
+                    {
+                        "type": "object",
+                        "description": "Nessus Scanner",
+                        "properties": {
+                            "server": {
+                                "type": "object",
+                                "properties": {
+                                    "url": {
+                                        "type": "string"
+                                    },
+                                    "username": {
+                                        "type": "string"
+                                    },
+                                    "password": {
+                                        "type": "string"
+                                    }
+                                },
+                                "required": [
+                                    "url",
+                                    "username",
+                                    "password"
+                                ]
+                            },
+                            "scan": {
+                                "type": "object",
+                                "properties": {
+                                    "name": {
+                                        "type": "string"
+                                    },
+                                    "folder": {
+                                        "type": "string"
+                                    },
+                                    "policy": {
+                                        "type": "string"
+                                    },
+                                    "timeout": {
+                                        "type": "integer"
+                                    },
+                                    "targets": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "string"
+                                        }
+                                    }
+                                },
+                                "required": [
+                                    "name",
+                                    "folder",
+                                    "targets"
+                                ]
+                            }
+                        },
+                        "required": [
+                            "scan",
+                            "server"
+                        ]
+                    }
+                ]
+            }
+        }
+    },
+    "required": [
+        "scanners"
```

Comment:
it seems like `scanners` is the only required property, though I'm not 100% sure

Reply:
config(version) if we agree to have it required.

Reply:
Added, since I've also added `configVersion`